### ISSUE TITLE
Can't find foundation-icons.3

### ISSUE THREAD

**Issue by alang9** [CUFP/cufp.org#71]
I don't seem to be able to find this dependency anywhere:

```
=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[ERROR] curl: code 404 while downloading http://solvuu.com/download/foundation-icons.3.tar.gz
[ERROR] The sources of the following couldn't be obtained, aborting:
          - foundation-icons.3
        (This may be fixed by running 'opam update')
```

**Comment by agarwal** [comment]
Thanks for reporting this. I never got a GitHub notification about it, so didn't see it till now.

This package and several others are maintained by me in the opam repository [opam-repo-web](https://github.com/solvuu/opam-repo-web), which mostly contains packages that are also in `npm`. Thus, I think the better solution is to start depending on `npm` instead of my half-baked `opam-repo-web`. Of course that'll create an extra dependency on the `npm` command line tool, but that seems okay given how widely used it is.

Some packages in `opam-repo-web`, such as the one causing the current issue, are not in `npm`. In these cases we can simply download them as part of the build script within this project. I'll update the Makefile to support this.

**Comment by agarwal** [comment]
Fixed in d236712d8fd532e315a2dc63973ed2b154fed8e5.
